Libplanet.Explorer/Libplanet.Explorer.csproj

```diff
@@ -26,7 +26,8 @@
         runtime; build; native; contentfiles; analyzers
       </IncludeAssets>
     </PackageReference>
-    <PackageReference Include="GraphQL" Version="2.4.0" />
+    <PackageReference Include="GraphQL" Version="3.0.0" />
```

Comment:
Wouldn't this upgrading change any public GraphQL APIs or behaviors?

https://graphql-dotnet.github.io/docs/guides/migration3#breaking-changes

Reply:
I read it 🙇
And I believe there are no features to update because it doesn't use many features of GraphQL.NET fully (e.g., subscription and Dependency Injection). I updated all changes at API-level. If we decided to use a pattern with Dependency Injection or to remove Newtonsoft Json dependency actively, it will have more features to look at in detail.

Reply:
If you don't have additional opinions on this thread, I will merge this pull request. If you didn't leave a comment yet though you have, please leave it. 🙏

Reply:
What I meant was: doesn't this upgrading change the result for any same input query from users?  For example, the new version apparently supports more types than its previous versions, so it might change representations of some fields having types that had been unsupported before 3.0.0?

Reply:
There were no such types at least when I saw.
Also, I added more tests to check that other changes will break compatibility as you said.

```
    Byte
    SByte
    Short
    UShort
    UInt
    ULong
    Guid (maps to IdGraphType by default; also supports GuidGraphType)
    BigInt
```